### PR TITLE
Added RelayState requirement for SAML 2.0

### PR DESCRIPTION
**Pull request by cwarnermm**
Updated documentation based on Customer Support feedback.

### CODE REVIEW

File: source/configure/authentication-configuration-settings.rst [hunk 1]
```diff
@@ -1184,8 +1184,10 @@ SAML 2.0
...
 
-.. note::
-   In line with Microsoft ADFS guidance we recommend `configuring intranet forms-based authentication for devices that do not support WIA <https://docs.microsoft.com/en-us/windows-server/identity/ad-fs/operations/configure-intranet-forms-based-authentication-for-devices-that-do-not-support-wia>`__.
+.. important::
+  
+  - When setting up and using SAML 2.0, ``relaystate`` must be passed back to the client from the identify provider. If you're integrating Mattermost with a provider that doesn't already do so by default, ensure that ``relaystate`` is enabled. See the PingIdentity `SAML RelayState <https://support.pingidentity.com/s/article/SAML-RelayState-and-PingFederate#:~:text=RelayState%20is%20an%20optional%20parameter,the%20SAML%202.0%20bindings%20specification>`_ documentation for details.
+  - In line with Microsoft ADFS guidance, we recommend `configuring intranet forms-based authentication for devices that do not support WIA <https://docs.microsoft.com/en-us/windows-server/identity/ad-fs/operations/configure-intranet-forms-based-authentication-for-devices-that-do-not-support-wia>`__.
```
Comment (hanzei) [inline source/configure/authentication-configuration-settings.rst:1190]: I'm a bit hesitant to add additional information to the config page. The `relaystate` requirement is more of a troubleshooting notice, as most of the time it works out of the box. I wonder if the notice in `source/onboard/sso-saml-technical.rst` is good enough.

Comment (hanzei) [inline source/configure/authentication-configuration-settings.rst:1190]: I'm a bit hesitant to add additional information to the config page. The `relaystate` requirement is more of a troubleshooting notice, as most of the time it works out of the box. I wonder if the notice in `source/onboard/sso-saml-technical.rst` is good enough.

Comment (cwarnermm) [reply source/configure/authentication-configuration-settings.rst:1190]: Thanks, @hanzei -- I'll remove the additional reference in the config section.

**Comment by cwarnermm** [comment]
@hanzei - I'm merging this docs PR into the v9.11 docs branch in preparation for tomorrow's product and docs release. I can apply any feedback you have to that branch or production directly if needed.

**Review by hanzei — COMMENTED** [review]
LGTM :+1:

**Pull request merged**